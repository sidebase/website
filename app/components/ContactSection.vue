<template>
  <div class="w-full bg-gray-800 py-12 px-4">
    <h1 class="text-4xl text-center text-gray-200">
      Contact us
    </h1>
    <div class="text-center text-gray-300 text-xl my-5">
      <p>
        Do you have any issues? Send us an email and we will respond as soon as possible!
      </p>
      <p class="mt-8">
        Alternatively, you can also reach us by telephone.
      </p>
    </div>
    <div class="flex flex-col md:flex-row items-center gap-3 w-full justify-center mt-8">
      <a href="mailto:sidebase@sidestream.tech" target="_blank">
        <Button label="Send us a message" class="p-button-outlined" icon="pi pi-envelope" icon-pos="left" />
      </a>
      <a href="https://github.com/sidestream-tech/sidebase/issues/new/choose" target="_blank">
        <Button label="Open an issue" icon="pi pi-github" icon-pos="left" />
      </a>
    </div>
  </div>
</template>
