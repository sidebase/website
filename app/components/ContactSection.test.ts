import { describe, expect, test } from 'vitest'
import ContactSection from './ContactSection.vue'
import { render } from '~/tests/testingLibraryVue'

describe('ContactSection', () => {
  test('mounts and is still the same', () => {
    const { html } = render(ContactSection)
    expect(html()).toMatchSnapshot()
  })
})
