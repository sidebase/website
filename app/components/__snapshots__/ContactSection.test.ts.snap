// Vitest Snapshot v1

exports[`ContactSection > mounts and is still the same 1`] = `
"<div class=\\"w-full bg-gray-800 py-12 px-4\\">
  <h1 class=\\"text-4xl text-center text-gray-200\\"> Contact us </h1>
  <div class=\\"text-center text-gray-300 text-xl my-5\\">
    <p> Do you have any issues? Send us an email and we will respond as soon as possible! </p>
    <p class=\\"mt-8\\"> Alternatively, you can also reach us by telephone. </p>
  </div>
  <div class=\\"flex flex-col md:flex-row items-center gap-3 w-full justify-center mt-8\\"><a href=\\"mailto:sidebase@sidestream.tech\\" target=\\"_blank\\"><button class=\\"p-button p-component p-button-outlined\\" type=\\"button\\">
        <!----><span class=\\"pi pi-envelope p-button-icon p-button-icon-left\\"></span><span class=\\"p-button-label\\">Send us a message</span>
        <!---->
      </button></a><a href=\\"https://github.com/sidestream-tech/sidebase/issues/new/choose\\" target=\\"_blank\\"><button class=\\"p-button p-component\\" type=\\"button\\">
        <!----><span class=\\"pi pi-github p-button-icon p-button-icon-left\\"></span><span class=\\"p-button-label\\">Open an issue</span>
        <!---->
      </button></a></div>
</div>"
`;
